src/scripts/blocks/countdown-timer/index.js

```diff
@@ -0,0 +1,66 @@
+import classnames from 'classnames';
+import DisplayComponent from './DisplayComponent';
+
+const { __ } = wp.i18n;
+const { registerBlockType } = wp.blocks;
+const { Component, Fragment } = wp.element;
+const {
+  InspectorControls, RichText,
+} = wp.editor;
+const {
+  PanelBody, Button, DateTimePicker, ColorPicker,
+} = wp.components;
+
+const { PostMediaSelector } = benenson.components;
+
+registerBlockType('benenson/block-countdown', {
+  title: __('Countdown Timer', 'benenson'),
+  description: __('Set a future date to countdown to.', 'benenson'),
+  icon: 'download',
```

Comment:
Please change the icon to [`clock`](https://developer.wordpress.org/resource/dashicons/#clock) or similar, as a download icon doesn't make that much sense in this scenario.